collections/incidents.coffee

```diff
@@ -67,4 +67,13 @@ if Meteor.isServer
           newIncident.date = incident.date
 
         Incidents.update(incident._id, {$unset: {deaths: ""}})
-        Incidents.insert(newIncident)
\ No newline at end of file
+        Incidents.insert(newIncident)
+
+    # Convert string case and death counts to integers
+    incidents = Incidents.find({$or: [{cases: {$type: "string"}}, {deaths: {$type: "string"}}]}).fetch()
+    for incident in incidents
+      if incident.cases
+        updateField = {cases: parseInt(incident.cases)}
+      else
+        updateField = {deaths: parseInt(incident.deaths)}
```

Comment:
Also, I think some error handling is needed in case parseInt doesn't return a number.